web/src/components/WikipediaBrowser/WikipediaBrowser.tsx

```diff
@@ -54,11 +54,19 @@ const WikipediaBrowser: React.FC = () => {
   const handleLinkClick = useCallback((e: React.MouseEvent<HTMLDivElement>, paneIndex: number) => {
     const target = e.target as HTMLElement;
     const link = target.closest("a");
-    if (link instanceof HTMLAnchorElement && link.href && link.title) {
+    const image = target.closest("img")
+    if(image){
```

Comment:
Can you add spaces here for correct formatting?

I will soon add a formatter to the project so that this can be done automatically.